### ISSUE TITLE
Table listing redesign

### ISSUE THREAD

**Issue by deontologician** [rethinkdb/rethinkdb#3825]
The current table listing is sub-optimal. It's grown a lot of buttons, and it's unnecessarily complicated to do some normal kinds of operations.

Case in point:

Deleting a table:

1. select the checkbox by the table
2. go to the top right and click "delete selected tables"
3. A modal pops up, asking you to confirm.
4. Click ok on the modal
5. the table disappears from the list
6. A success notification pops up saying you deleted the table.
7. Click the X on the notification to get rid of it.

What should happen:

1. Click a "delete table" icon on the table row.
2. Some small confirmation action (no modal)
3. The table disappears from the list.

We should have similarly easy actions for stuff like:

- renaming a table
- creating a new table/database

The big buttons aren't great either, we need some icons for them to make them stand out. Right now there's just a ton of huge grey buttons with nothing really distinguishing them.

**Comment by mglukhovsky** [comment]
Agreed. The biggest change we can make is inline actions (e.g. renaming happens inline, deletions happen inline, etc.). This would require creating some visual patterns, but it'll cut down on the chrome and set of steps you have to go through for basic tasks.

**Comment by danielmewes** [comment]
That sounds great. I don't think modals are particularly terrible, but I wouldn't be sad to see some of them go away either.
Adding small icons for performing specific actions to each individual row would be great, as would adding icons to the large buttons.

**Comment by deontologician** [comment]
#708 is relevant, the ui suffers a bit because there are so many indistinguishable buttons

**Comment by marshall007** [comment]
One thing I would really like to see added to the tables view is aggregated secondary index construction stats/progress.

**Comment by deontologician** [comment]
As part of this, we should allow renaming databases again
